manifoldco/manifold-subscription
# subscription list preview mode

## Change

<a href="https://www.loom.com/share/3c7573129e904d43b62c7f1c3aa1f3a2"> <p>Subscription List Preview Mode - Watch Video</p> <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/3c7573129e904d43b62c7f1c3aa1f3a2-with-play.gif"> </a>

## Testing

- add preview flag and product Id to manifold-subscription-list. (no auth token provided to manifold init)
  - the component should load up sample data
- add an auth token, and remove the product id and preview flag.
  - the component should load subscriptions for the specified owner
